Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml

```diff
@@ -272,3 +272,11 @@
   - type: Sprite
     sprite: Clothing/Neck/Cloaks/pan.rsi
 
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckCloakSynd
+  name: Syndicate cloak
+  description: dark like a crushed station. Nicely!
```

Comment:
This description makes no sense and the first letter should be capitalised so in this case it would be "Dark"